ClavaLaraExtraApi/src-java/pt/up/fe/specs/clava/extra/ClavaMathExtra.java

```diff
@@ -1,50 +0,0 @@
-/**
```

Comment:
Keep file

Reply:
Resolved in 44508bf26 with the creation of the `ClavaLaraExtraAPIs` branch.